.bumpversion.cfg

```
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 4.3.8
+current_version = 4.3.9
```

```suggestion
current_version = 4.3.8
```

I'll bump this after the merge